spec/features/fill_in_respondents_details_page_spec.rb

```diff
@@ -62,5 +64,6 @@
     respondents_details_page.contact_preference_question.assert_answers_for(user)
     expect(respondents_details_page.organisation_employ_gb_question.field.value).to eql user.organisation_employ_gb
     respondents_details_page.organisation_more_than_one_site_question.assert_answers_for(user)
+    respondents_details_page.video_call_question.set(user)
```

Comment:
You are setting the question with the user, not the user.video_call as line 25 does